directives/btn-group/btn-group.component.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    'use strict';
+
+    angular.module('wfm.btnGroup', [])
+    .component('wfmBtnGroup',
+    {
+        templateUrl: 'directives/btn-group/btn-group.tpl.html',
+        controller: BtnGroupCtrl,
+        bindings: {
+            items: '<',
+            selected: '=',
```

Comment:
how about using a `&` binding here?

Reply:
for the selected? 
its used both to get the selected item and set it to default.
for the items, sure

Reply:
yes, `selected`

if using a `&`, it will be in line with the recommendation from angular's docs.

Reply:
![image](https://user-images.githubusercontent.com/5781286/32095090-30d068e0-bb02-11e7-994b-43b65dde1551.png)

Would you prefer to have one input (selected) and one output binding?

Reply:
Using `=`, how could I know it when the `selected` is changed? Do I need to set up a watch to do that? As in Teams, we switch to the Day/Week view immediately when the selected button is changed.

Reply:
hmm.. no, so then we would need a callback on click